editor/components/TableMap.tsx

```diff
@@ -45,6 +47,12 @@ interface TableMapProps {
  * Map which can be configured to render in the Table component
  */
 export const TableMap = ({ mapRef, geojson, mapConfig }: TableMapProps) => {
+  const [basemapType, setBasemapType] = useState<"streets" | "aerial">(
+    "streets"
+  );
+
+  const basemapURL = useBasemap(basemapType);
```

Comment:
Doesn't change how this works at all, but it could be nice to package up the `basemap` state in the `useBasemap` custom hook since it will likely be needed for any map with dynamic basemaps.